A.html

```diff
@@ -0,0 +1,2 @@
+
+<h1>Hello Microverse!</h1>
```

Comment:
- [ ] Each HTML page must include a reference to the external style sheet file inside the element, inside the head section. Kindly add the HTML structure including a link to the CSS file.


for example
```

<!DOCTYPE html>
<html lang="en">
<head>
    <title>Microverse</title>
    <link rel="stylesheet" href="A.css">
</head>
<body>
    <h1>Hello Microverse!</h1>
</body>
</html>
```